tus_storagehandler/app.py

```diff
@@ -0,0 +1,57 @@
+"""API server entry point."""
+
+import logging
+import os
+from pathlib import Path
+
+from connexion import FlaskApp  # type: ignore
+from dotenv import load_dotenv  # type: ignore
+from foca import Foca
+
+load_dotenv()
+logger = logging.getLogger(__name__)
+
+
+def init_app() -> FlaskApp:
+    """Initialize and return the FOCA app.
+
+    This function initializes the FOCA app by loading the configuration
+    from the environment variable `TUS_FOCA_CONFIG_PATH` if set, or from
+    the default path if not. It raises a `FileNotFoundError` if the
+    configuration file is not found.
```

Comment:
Please document these variables in `README.md`, maybe create a diff section called Environment Variable in TOC and then add and define: 
- TUS_FOCA_CONFIG_PATH
- ENV (mentioned below)